### ISSUE TITLE
ci.centos.org is failing for "minishift openshift version" command

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#2041]
CI tests are failing with below error. Refer https://ci.centos.org/job/minishift-pr/3473/consoleFull
It seems that the stdout contains more letters than expected i.e. `openshift v3.7.1+a8deba5-34`
```console
06:56:17     Then stdout should match                                            # testsuite.go:573 -> /testsuite.commandReturnShouldMatchRegexContent
06:56:17       """
06:56:17       ^openshift v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{7}
06:56:17       kubernetes v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{10}
06:56:17       etcd [0-9]+\.[0-9]+\.[0-9]+
06:56:17       """
06:56:17     Output did not match. Expected: '^openshift v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{7}
06:56:17 kubernetes v[0-9]+\.[0-9]+\.[0-9]+\+[0-9a-z]{10}
06:56:17 etcd [0-9]+\.[0-9]+\.[0-9]+', Actual: 'openshift v3.7.1+a8deba5-34
06:56:17 kubernetes v1.7.6+a08f5eeb62
06:56:17 etcd 3.2.8
06:56:17 '

****
```

### Expected

```
$ minishift openshift version
openshift v3.7.1+a8deba5
kubernetes v1.7.6+a08f5eeb62
etcd 3.2.8
```
### Actual

```
$ minishift openshift version
openshift v3.7.1+a8deba5-34
kubernetes v1.7.6+a08f5eeb62
etcd 3.2.8
```

**Comment by LalatenduMohanty** [comment]
I could reproduce this locally after turning off the image caching i.e. `$ minishift config set image-caching false`. Looks like OpenShift team has pushed new images for v3.7.1 and the version has changed. 

```
$ minishift openshift version
openshift v3.7.1+a8deba5-34
kubernetes v1.7.6+a08f5eeb62
etcd 3.2.8
```

**Comment by praveenkumar** [comment]
https://regex101.com/r/ZfFsbm/1 @agajdosi @LalatenduMohanty I hope this regex will make sense because then we are compatible with the older version also.

**Comment by LalatenduMohanty** [comment]
@praveenkumar Yes, looks good to me.

**Comment by agajdosi** [comment]
@praveenkumar  +1

**Comment by LalatenduMohanty** [comment]
Resolved via #2043
